pipeline/metadata/design_table.csv

```diff
@@ -48,11 +48,14 @@ Compass_PTR_UP_313,TerosTableA,PB,PTR,UP,TEROS12,,,,,,
 Compass_PTR_UP_313,TerosTableA,Format,PTR,UP,TEROS12,,,,,,
 Compass_PTR_UP_313,TerosTableA,Statname,PTR,UP,TEROS12,,,,,,
 Compass_PTR_UP_313,TerosTableA,BattV_Avg,PTR,UP,TEROS12,A,,,,battery_voltage,
-Compass_PTR_UP_313,TerosTableA,"TerosA({1:4},1)",PTR,UP,TEROS12,A,{1:4},,,soil_vwc_10cm,
+Compass_PTR_UP_313,TerosTableA,"TerosA({1:2},1)",PTR,UP,TEROS12,A,{1:2},,,soil_vwc_10cm,
+Compass_PTR_UP_313,TerosTableA,"TerosA({3:4},1)",PTR,UP,TEROS12,A,{3:4},,,,
 Compass_PTR_UP_313,TerosTableA,"TerosA({5:6},1)",PTR,UP,TEROS12,A,{5:6},,,soil_vwc_30cm,
-Compass_PTR_UP_313,TerosTableA,"TerosA({1:4},2)",PTR,UP,TEROS12,A,{1:4},,,soil_temp_10cm,
+Compass_PTR_UP_313,TerosTableA,"TerosA({1:2},2)",PTR,UP,TEROS12,A,{1:2},,,soil_temp_10cm,
+Compass_PTR_UP_313,TerosTableA,"TerosA({3:4},2)",PTR,UP,TEROS12,A,{3:4},,,soil_temp_10cm,
```

Comment:
This extra row necessary re: above?

Reply:
No idea where my head was when I made these changes, sorry

Reply:
the post-holiday fog!!